### PR TITLE
Monkey patches certifi.where() to load OS mounted CA path (KO-2744)

### PR DESCRIPTION
**Pull request by mikeatlas-r7**
See code comments.

This depends on komand hotfix/0.30.1 - https://github.com/komand/komand/pull/1702

### CODE REVIEW

**Comment by mrinehart-r7** [comment]
Looks good to me @mikeatlas-r7

**Comment by mikeatlas-r7** [comment]
@mrinehart-r7 can you mark this PR as passed review? I'll do the python2 branch separately after this.

**Comment by mikeatlas-r7** [comment]
also @mrinehart-r7 @mbroomfield-r7 @jschipp-r7 - just fyi, this is about as far as I can take this to get python plugins to pick up internal CA certificates from the host mounted bundle - from here I'll need to defer to you guys to rebuild any other plugins which aren't using `requests` or `certifi` library (e.g., imap or exchange plugins - the protocol may be TLS and need SSLContexts, but won't be HTTPS).

File: komand/__init__.py [hunk 1]
```diff
@@ -14,3 +14,33 @@
...
+            # and return callers of certifi.where() a shiny custom path
+            # to do their own verifications against the bundle.
+            return env_var
+        else:
+            return old_certifi_value
+    except Exception as ex:
+        return old_certifi_value
+
+# and here's the monkey-patch itself.
+certifi.where = where
```
Comment (mbroomfield-r7) [inline komand/__init__.py]: Can you add a newline here for PEP8 compliance?

**Pull request merged**